indico/modules/events/templates/display/_details.html

```diff
@@ -0,0 +1,84 @@
+{% from 'events/timetable/display/_common.html' import render_speakers, render_references,
+                                                       render_description %}
+{% from 'attachments/_display.html' import render_attachments_folders %}
+
+<table class="eventDetails" id="eventDetails">
+<tbody>
+    {% if conf.getDescription() %}
+        <tr id="eventDescription">
+            <td class="leftCol">
+                {%- trans %}Description{% endtrans -%}
+            </td>
+            <td>{{ render_description(conf) }}</td>
+        </tr>
+    {% endif %}
+
+    {% if event.references and conf.getType() == "meeting" %}
```

Comment:
Would it make sense to move the event type into the new DB instead of using the weird logic we currently have (checking the webfactory)? (not in this PR of course, but in general)